Modules/Microsoft365DSC/SchemaDefinition.json

```diff
@@ -3519,6 +3519,81 @@
                            }
                        ]
     },
+    {
+        "ClassName":  "MSFT_AADFeatureRolloutPolicy",
```

Comment:
In the future, we don't need to touch this file. It gets generated automatically upon merging the PR.